src/osd/PG.h

```diff
@@ -1457,10 +1457,10 @@ class PG : public DoutPrefixProvider,
  */
 class PGLockWrapper {
  public:
-  explicit PGLockWrapper(PGRef locked_pg) : m_pg{locked_pg} {}
+  explicit PGLockWrapper(PGRef locked_pg) : m_pg{std::move(locked_pg)} {}
```

Comment:
Probably simpler to template the argument and use std::forward.

Reply:
Modified as suggested.